packages/sitecore-jss-react/src/components/EditingScripts.test.tsx

```diff
@@ -75,31 +76,62 @@ describe('<EditingScripts />', () => {
     expect(scripts.find('script')).to.have.length(0);
   });
 
-  ['Preview', 'Edit'].forEach((pageState) => {
-    it(`should render nothing when ${pageState} and edit mode is Chromes`, () => {
-      const layoutData = getLayoutData({
-        editMode: EditMode.Chromes,
-        pageState: LayoutServicePageState[pageState],
-        pageEditing: true,
-      });
+  it('should render nothing when Preview and edit mode is Chromes', () => {
```

Comment:
how is this testing Edit mode?

Reply:
It doesn't. It tests preview mode, when Pages is in Chromes mode.